nilearn/plotting/find_cuts.py

```diff
@@ -337,3 +337,115 @@ def find_cut_slices(img, direction='z', n_cuts=7, spacing='auto'):
     cut_coords.sort()
 
     return _transform_cut_coords(cut_coords, direction, affine)
+
+
+def find_parcellation_cut_coords(labels_img, background_label=0, return_label_names=False,
+                                 label_hemisphere='left'):
+    """ Return coordinates of center of mass of 3D parcellation atlas
+
+    Parameters
+    ----------
+    labels_img: 3D Nifti1Image
+        A brain parcellation atlas with specific mask labels for each
+        parcellated region.
+
+    background_label: int, optional (default 0)
+        Label value used in labels_img to represent background.
+
+    return_label_names: bool, optional (default False)
+        Returns list of labels
+
+    label_hemisphere: 'left' or 'right', optional (default 'left')
+        Choice of hemisphere to compute label center coords for.
+        Applies only in cases where atlas labels are lateralized.
+        Eg. Yeo or Harvard Oxford atlas.
+
+    Returns
+    -------
+    coords: numpy.ndarray of shape (n_labels, 3)
+        Label regions cut coordinates in image space (mm).
+
+    labels_list: list, optional
+        Label region. Returned only when return_label_names is True.
+
+    See Also
+    --------
+    nilearn.plotting.find_probabilistic_atlas_cut_coords : For coordinates
+        extraction on probabilistic atlases (4D) (Eg. MSDL atlas)
+    """
```

Comment:
We should check that label_hemisphere is in 'right', 'left', and raise a meaningful error here.